sceptre/scipool/templates/sc-tag-options.j2

```diff
@@ -8,7 +8,7 @@ Resources:
     Properties:
       Active: true
       Key: "CostCenter"
-      Value: {{ CostCenter }}
+      Value: {{ CostCenter.replace('(','').replace(')','') }}
```

Comment:
I'm not sure I like this idea because the tag that gets applied to resources will not match the tags from the source of truth.  This may cause problems for tools that query and make use of these tags.

Reply:
good point, I hadn't thought of that. we need to update the source of truth for all of the tooling, so I guess the characters should be removed from the tag name in `lambda-mips-api`